mono/mini/interp/interp.c

```diff
@@ -750,7 +750,12 @@ static InterpMethodArguments* build_args_from_sig (MonoMethodSignature *sig, Mon
 			break;
 #if SIZEOF_VOID_P == 4
 		case MONO_TYPE_I8:
-			g_assert (0);
+#ifdef TARGET_ARM
+			/* pairs begin at even registers */
+			if (margs->ilen & 1)
+				margs->ilen++;
+#endif
```

Comment:
thanks!  I don't have a device that runs `!eabi`, is the following okay for now? https://github.com/mono/mono/pull/4773/commits/033c4dd6eb5b566610961a453af60a154d9ded60